src/com/oltpbenchmark/OLTPBench.java

```diff
@@ -0,0 +1,190 @@
+            options.addOption("o", "output", true, "Output file (default System.out)");
+            options.addOption("d", "directory", true, "Base directory for the result files, default is current directory");
+            options.addOption("t", "timestamp", false, "Each result file is prepended with a timestamp for the beginning of the experiment");
+            options.addOption("ts", "tracescript", true, "Script of transactions to execute");
+            options.addOption((String)null, "histograms", false, "Print txn histograms");
+            options.addOption((String)null, "dialects-export", true, "Export benchmark SQL to a dialects file");
+            CommandLine argsLine = parser.parse(options, args);
+            if (argsLine.hasOption("h")) {
+                oltpMethods.printUsage(options);
+            } else if (!argsLine.hasOption("c")) {
+                LOG.error("Missing Configuration file");
+                oltpMethods.printUsage(options);
+            } else if (!argsLine.hasOption("b")) {
+                LOG.fatal("Missing Benchmark Class to load");
+                oltpMethods.printUsage(options);
+            } else {
+                int intervalMonitor = 0;
+                if (argsLine.hasOption("im")) {
+                    intervalMonitor = Integer.parseInt(argsLine.getOptionValue("im"));
+                }
+
+                String targetBenchmarks = argsLine.getOptionValue("b");
+                String[] targetList = targetBenchmarks.split(",");
+                List<BenchmarkModule> benchList = new ArrayList();
+                List<TransactionType> activeTXTypes = new ArrayList();
+                String configFile = argsLine.getOptionValue("c");
+                XMLConfiguration xmlConfig = new XMLConfiguration(configFile);
+                xmlConfig.setExpressionEngine(new XPathExpressionEngine());
+                System.out.println(oltpMethods.pluginConfigBenchmark(targetList, xmlConfig, argsLine, pluginConfig, configFile, activeTXTypes, benchList));
+
+                assert !benchList.isEmpty();
+
+                assert benchList.get(0) != null;
+
+                if (oltpMethods.isBooleanOptionSet(argsLine, "dialects-export")) {
+                    BenchmarkModule bench = (BenchmarkModule)benchList.get(0);
+                    if (bench.getStatementDialects() != null) {
+                        LOG.info("Exporting StatementDialects for " + bench);
+                        String xml = bench.getStatementDialects().export(bench.getWorkloadConfiguration().getDBType(), bench.getProcedures().values());
+                        System.out.println(xml);
+                        System.exit(0);
+                    }
+
+                    throw new RuntimeException("No StatementDialects is available for " + bench);
+                } else {
+                    boolean verbose = argsLine.hasOption("v");
+                    Iterator var15;
```

Comment:
Why did you use the name `var15`?

Reply:
Actually I had renamed class name 'DBWorkload' to 'OLTPBench' just and I hadn't refactoring inside this code with any changes of variables names or methods. Unfortunately I don't understand how var 15  was appeared, because I wasn't creating it. It is a bit strange however I'am trying to figure out causes of appearings these variables.